src/Microsoft.Identity.Web/DownstreamWebApiSupport/MicrosoftIdentityAuthenticationMessageHandlerOptions.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Identity.Web
+{
+    /// <summary>
+    /// Options passed-in to Microsoft Identity message handlers.
+    /// </summary>
+    public class MicrosoftIdentityAuthenticationMessageHandlerOptions : ICloneable

```

Comment:
@qetza why not use the TokenAcquisitionOptions directly (maybe inheriting from them) ?

Reply:
@jmprieur, i think you mean reusing `DownstreamWebApiOptions`. This class effectively is a subset of `DownstreamWebApiOptions`, i could make `DownstreamWebApiOptions` inherit from this one and add its specific properties.